sigma/scheduledqueryrun/client_test.go

```diff
@@ -0,0 +1,28 @@
+package scheduledqueryrun
+
+import (
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+	stripe "github.com/stripe/stripe-go"
+	_ "github.com/stripe/stripe-go/testing"
+)
+
+func TestSigmaScheduledQueryRunGet(t *testing.T) {
+	t.Skip("skipping test as stripe-mock does not support this resource")
```

Comment:
It would be really cool if we could add some way to stub requests like we do in other libraries!

Reply:
Yes, that would be cool! Mocking isn't really possible in Go, but I think we can accomplish something similar with a helper to boot up a little JSON test server that responds with a blob you can hand it.

In this case though, I think we should make sure that stripe-mock supports scheduled query runs. I'm just winding that through the backend now.

Reply:
Okay, can you try bumping stripe-mock to 0.24.0 and seeing if you can activate these tests? Thanks!

ptal @remi-stripe

Reply:
oh neat, thanks for shipping this one!